pkg/stringutils/stringutils_test.go

```diff
@@ -60,17 +60,32 @@ func TestGenerateRandomAsciiStringIsAscii(t *testing.T) {
 func TestEllipsis(t *testing.T) {
 	str := "t🐳ststring"
 	newstr := Ellipsis(str, 3)
-	if newstr != "t🐳s" {
-		t.Fatalf("Expected t🐳s, got %s", newstr)
+	if newstr != "t🐳" {
```

Comment:
Is this expected? I don't think the whale emoji is a wide character?

Reply:
The character do takes two positions of a narrow one. And the test will pass only in this way, after we adopt https://godoc.org/golang.org/x/text/width

Reply:
Ah! Yes, you're right, I thought it only took a single width, but I see now.

Sorry for the noise 😊